src/platform/test/functional/apps/discover/ccs_compatibility/_saved_queries.ts

```diff
@@ -270,6 +270,37 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await queryBar.switchQueryLanguage('lucene');
         expect(await queryBar.getQueryString()).to.eql('');
       });
+
+      it('checks if the "Save query" button becomes enabled after adding filters, even when the query was saved with "Include filters" unchecked', async () => {
+        await queryBar.setQuery('response:200');
+        await queryBar.submitQuery();
+        await savedQueryManagementComponent.saveNewQuery('filterExcluded', '', false, false);
+        await savedQueryManagementComponent.savedQueryExistOrFail('filterExcluded');
+        await savedQueryManagementComponent.closeSavedQueryManagementComponent();
+        await savedQueryManagementComponent.openSavedQueryManagementComponent();
+        let isSaveButtonDisabled = await testSubjects.getAttribute(
+          'saved-query-management-save-button',
+          'disabled'
+        );
+        if (!isSaveButtonDisabled) {
+          throw new Error('Save button should be disabled directly after saving a query');
```

Comment:
Nit: `expect` calls should be used for these assertions. You can pass custom messages as the second param to the assertion methods if you'd like to customize them, e.g.
```ts
expect(isSaveButtonDisabled).to.equal(true, 'Save button should be disabled directly after saving a query');
```